app/views/facility_account_orders/index.html.haml

```diff
@@ -31,31 +31,29 @@
 
           %td= order_detail.ordered_at
           %td
-            %ul
-              %li= order_detail.description_as_html
-              - if order_detail.reservation.present?
-                - if order_detail.reservation_admin_editable?
-                  %li.indented
-                    = link_to order_detail.reservation,
-                      order_detail.edit_reservation_path
-                - else
-                  %li.indented
-                    = link_to order_detail.reservation,
-                      order_detail.show_reservation_path
+            = order_detail.description_as_html
+            - if order_detail.reservation.present?
+              - if order_detail.reservation_admin_editable?
+                .indented
+                  = link_to order_detail.reservation,
+                    order_detail.edit_reservation_path
+              - else
+                .indented
+                  = link_to order_detail.reservation,
+                    order_detail.show_reservation_path
 
-              - if order_detail.survey_completed?
-                %li.indented
-                  = link_to t(".label.view_order.form"),
-                    order_detail.survey_url, target: "_blank"
+            - if order_detail.survey_completed?
+              .indented
+                = link_to t(".label.view_order.form"),
+                  order_detail.survey_url, target: "_blank"
 
-              - if order_detail.stored_files_template_result.present?
-                - order_detail.stored_files_template_result.each do |stored_file|
-                  %li.indented
-                    = link_to t(".label.view_order.file"), stored_file.download_url
+            - if order_detail.stored_files_template_result.present?
+              - order_detail.stored_files_template_result.each do |stored_file|
+                .indented
+                  = link_to t(".label.view_order.file"), stored_file.download_url
 
-              - if order_detail.note.present?
-                %li
-                  %i= t(".note", text: order_detail.note)
+            - if order_detail.note.present?
+              = render partial: "shared/order_detail_note", locals: { order_detail: order_detail }
```

Comment:
I think we're generally moving towards `render "shared/order_detail_note", order_detail: order_detail`, but it's not a huge deal.